README.md

```diff
@@ -10,6 +10,8 @@ different environments, including:
 *   [Cloud Run and Cloud Run on GKE](https://cloud.google.com/run/)
 *   [Knative](https://github.com/knative/)-based environments
 
+These functions allow you to easily handle **HTTP requests** or **[CloudEvents](https://cloudevents.io/)** in a FaaS programming style.
```

Comment:
Maybe first mention of FaaS should be `Functions as a Service (FaaS)`?

Reply:
Added.